apps/documentation/docs/fresco/deployment/guide.en.mdx

```diff
@@ -1,96 +1,79 @@
 
-- **GitHub**: A platform for hosting and sharing code. The Fresco code is shared via GitHub, and you will need a GitHub account to deploy it.
-- **Node.js**: A JavaScript runtime that allows you to run JavaScript code outside of a web browser. Fresco uses Node.js to run the server that hosts the application.
-- **PostgreSQL**: A powerful, open-source relational database system. Fresco uses PostgreSQL to store data about your study, participants, and responses.
-- **UploadThing**: A cloud-based file storage service that Fresco uses to store media files, such as images and audio recordings.
+It is also possible to use whatever infrastructure you may have access to in order to deploy Fresco. For information on this, see our [advanced deployment](./advanced) guide.
 
-To make the deployment process as simple as possible, we designed Fresco to be easily deployed on
-the [Vercel](https://vercel.com) platform, which provides both the Node.js and the PostgreSQL
-services for you (for free).
+## Create fork of Fresco
 
-Vercel is a cloud platform that specializes in deploying web
-applications. It is designed to be easy to use and has a generous free tier that is suitable for
-most research projects. It is not a requirement to use Vercel, but it is the simplest way to get
-started, and is the method we will be using in this guide.
+Deploying Fresco involves setting up your own unique version of the project, which we refer to as your **instance**. The first step in creasing this insance is to create a copy (or "fork") of the Fresco code on GitHub.
 
-## Initial Deployment to Vercel
+To do this, ensure you have a GitHub account and are signed in, and then:
 
-Deploying Fresco to Vercel involves setting up your own unique version of the project, which we
-refer to as your **instance**. This instance will be hosted on Vercel's servers and will be
-accessible via a unique URL. You will be able to access your instance of Fresco from any web browser
-by visiting this URL.
+1. Visit the  <a href="https://github.com/complexdatacollective/Fresco" target="_blank">Fresco repository on GitHub</a>, and click **"Fork"**.
 
-<Tipbox>
-
-Should you run into any issues, please consult the [troubleshooting](/en/fresco/deployment/troubleshooting) guide or reach out on the [user community](https://community.networkcanvas.com).
+   ![](/assets/img/fresco-images/github-fork1.png)
 
-</Tipbox>
+2. Enter a repository name (this will be your Fresco instance name, so you may wish to use your study name or the name of your project), along with a description, and click **"Create Fork"**. Leave the "copy main branch" option selected.
+   ![](/assets/img/fresco-images/github-fork2.png)
```

Comment:
There should be a space between the text and the image on lines 65-66, because they are stuck together in view: https://documentation-r7yz8y5zk-network-canvas-f4790d84.vercel.app/en/fresco/deployment/guide

Reply:
fixed